janno_columns.tsv

```diff
@@ -1,37 +1,37 @@
-Date_Type	“C14“ if directly from the individual, “contextual“ if based on archaeology or other C14 dates from the site, “modern” for present-day individuals	String choice	C14;contextual;modern	FALSE	FALSE		
-No_of_Libraries	number of libraries	Integer		FALSE	FALSE		
-Data_Type	specifics of data generation method	String choice	Shotgun;1240K;OtherCapture;ReferenceGenome	FALSE	FALSE		
-Genotype_Ploidy	ploidy of the genotypes	String choice	diploid;haploid	FALSE	FALSE		
-Group_Name	ideally Eisenmann rule + underscore flags, e.g. to annotate relatives or outliers or low coverage, multiple entries separated by ; to accommodate different labels, value must equal the group name in the .fam file (in case of multiple entries the first one)	String list		TRUE	FALSE		
-Genetic_Sex	“F“, “M“ or “U“ because eigenstrat and plink formats only support these three. Edge cases (XXY, XYY, X0) are undefined and should be grouped as F, M or U, with a note added	Char choice	F;M;U	TRUE	FALSE		
-Nr_autosomal_SNPs	number of autosomal SNPs covered for 1240K capture or SG data pulldown	Integer		FALSE	FALSE		
-Coverage_1240K	average X-fold coverage across 1240K SNP sites after quality filtering (internal data), NOT the % SNPs of 1.2M possible	Float		FALSE	FALSE		
-MT_Haplogroup	mitochondrial haplogroup after phylotree.org as reported by Haplofind or Haplogrep	String		FALSE	FALSE		
-Y_Haplogroup	Y-chromosome haplogroup reported as published, for internal data, please follow syntax with main branch + most terminal derived Y-SNP (e.g. R1b-P312)	String		FALSE	FALSE		
-Endogenous	% endogenous DNA as estimated from SG libraries (before capture), as for example estimated by EAGER for the best library (in percent), not on target and no quality filter	Float		FALSE	FALSE	0	100
-UDG 	“mixed” in case multiple libraries with different UDG treatment were merged	String choice	minus;half;plus;mixed	FALSE	FALSE		
-Library_Built	“ds” for double stranded, “ss” for single stranded	String choice	ds;ss;other	FALSE	FALSE		
-Damage 	% damage on 5' end for the main shotgun library used for sequencing and/or capture	Float		FALSE	FALSE	0	100
-Xcontam 	if male for captured library	Float		FALSE	FALSE	0	1
-Xcontam_stderr	standard error of ANGSD X contamination estimate	Float		FALSE	FALSE	0	Inf
-mtContam	mitochondrial contamination rate as estimated by ContamMix and/or Schmutzi	Float		FALSE	FALSE	0	1
-mtContam_stderr	Standard error of ContamMix/Schmutzi estimate	Float		FALSE	FALSE	0	Inf
-Primary_Contact	Project lead or first author	String		FALSE	FALSE		
-Publication_Status	bibtex key (e.g. “AuthorJournalYear“) or “unpublished“	String		FALSE	FALSE		
-Note	wildcard comments. e.g. note down aneuploidies here	String		FALSE	FALSE		
-Keywords	Arbitrary tags separated by ; (e.g. for custom sorting purposes) 	String list		FALSE	FALSE		
+janno_column_name	description	data_type	multi	choice	range	choice_options	range_lower	range_upper	mandatory	unique
```

Comment:
That makes sense, extending the columns for quick Boolean checks.

Reply:
The rest looks all good. I have a bit of a hard time reviewing every row here, because of the column title changes and additions. But I guess it looks all fine, and it includes the new multi-options for the library properties. So good to go.

Reply:
In this case reviewing the finished product (https://github.com/poseidon-framework/poseidon2-schema/blob/multi_library/janno_columns.tsv) is probably the most easy way. 

Anyway: Thanks for taking a look! If @AyGhal and @wolfgangaroo confirm this, then we can go forward quickly.